style.css

```diff
@@ -0,0 +1,1285 @@
+    width: 50%;
+  }
+}
+
+@media (max-width: 530px) {
+  .first-page {
+    background: url(./Img/mainpic-mob.png) no-repeat bottom;
+    background-size: contain;
+    width: 100%;
+    min-height: 844px;
+  }
+
+  .header {
+    height: 98px;
+  }
+
+  .header__logo {
+    padding: 20px 0 0 37px;
+  }
+
+  .header__logo-loco {
+    font-size: 42px;
+    letter-spacing: 3.5px;
+  }
+
+  .header__menu {
+    width: 98px;
+    height: 98px;
+  }
+
+  .header__menu-icon {
+    height: 27px;
+  }
+
+  .header__menu-line {
+    width: 38px;
+    margin: 6px 29px;
+  }
+
+  .wrap__content {
+    background: none;
+    height: auto;
+  }
+
+  .wrap__heading {
+    padding: 0 0 0 35px;
+    width: auto;
+    font-size: 71px;
+    line-height: 73px;
+    letter-spacing: -.1px;
+    margin-top: 512px;
+  }
+
+  .wrap__heading::after {
+    margin-left: 44%;
+    width: 26px;
+    height: 26px;
+    margin-top: 19px;
+  }
+
+  .second-page,
+  .third-page {
+    font-size: 9px;
+  }
+}
+
+@media (max-width: 480px) {
+  .first-page {
+    min-height: 765px;
+  }
+
+  .header {
+    height: 89px;
+  }
+
+  .header__logo {
+    padding: 19px 0 0 33px;
+  }
+
+  .header__logo-loco {
+    font-size: 38px;
+  }
+
+  .header__menu {
+    width: 90px;
+    height: 90px;
+  }
+
+  .header__menu-icon {
+    height: 25px;
+  }
+
+  .header__menu-line {
+    width: 34px;
+    margin: 6px 28px;
+  }
+
+  .wrap__heading {
+    padding: 0 0 0 32px;
+    font-size: 64px;
+    line-height: 67px;
+    margin-top: 464px;
+  }
+
+  .wrap__heading::after {
+    margin-left: 44%;
+    width: 24px;
+    height: 24px;
+    margin-top: 16px;
+  }
+}
+
+@media (max-width: 440px) {
+  .first-page {
+    min-height: 685px;
+  }
+
+  .header {
+    height: 75px;
+  }
+
+  .header__logo {
+    padding: 15px 0 0 29px;
+  }
+
+  .header__logo-loco {
+    font-size: 33px;
+    letter-spacing: 2.4px;
+  }
+
+  .header__menu {
+    width: 74px;
+    height: 74px;
+  }
+
+  .header__menu-icon {
+    height: 21px;
+  }
+
+  .header__menu-line {
+    width: 29px;
+    height: 3px;
+    margin: 5px 22px;
+  }
+
+  .wrap__heading {
+    padding: 0 0 0 28px;
+    font-size: 54px;
+    line-height: 57px;
+    letter-spacing: .3px;
+    margin-top: 396px;
+  }
+
+  .wrap__heading::after {
+    margin-left: 43%;
+    margin-top: 11px;
+    width: 22px;
+    height: 22px;
+    border-right: 4px solid #000;
+    border-top: 4px solid #000;
+  }
+}
+
+@media (max-width: 400px) {
+  .first-page {
+    min-height: 636px;
+  }
+
+  .wrap__heading {
+    letter-spacing: -.5px;
+    margin-top: 384px;
+  }
+}
+
+@media (max-width: 384px) {
+  .wrap__heading {
+    font-size: 50px;
+  }
+}
+
+@media (max-width: 320px) {
+  .first-page {
+    min-height: 573px;
+  }
+
+  .header {
+    height: 67px;
+  }
+
+  .header__logo {
+    padding: 13px 0 0 25px;
+  }
+
+  .header__logo-loco {
+    font-size: 29px;
+    letter-spacing: 2px;
+  }
+
+  .header__menu {
+    width: 67px;
+    height: 67px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633356)